lib/util.js

```diff
@@ -10,3 +13,27 @@ exports.readPkg = () => {
 
 	return pkg;
 };
+
+exports.linkifyIssues = (url, message) => {
+	if (!supportsHyperlinks.stdout) {
```

Comment:
I didn't experience any problems with leaving this out though. But maybe Windows renders the control characters...